quiche/src/lib.rs

```diff
@@ -4001,15 +3996,14 @@ impl Connection {
             path.active() &&
             !dgram_emitted
         {
-            while let Some(stream_id) = self.streams.peek_flushable() {
+            for stream_id in self.streams.flushable() {
```

Comment:
For the purposes of this code we don't actually need the full iterator since we only end up using only one stream anyway, so I think we should keep the `peek_flushable()` API, which should be easy to implement on top of intrusive rbtree I think.

Reply:
Sure